cabal-testsuite/PackageTests/ReplBuildDepends/cabal.test.hs

```diff
@@ -0,0 +1,35 @@
+import Test.Cabal.Prelude
+
+-- Test “repl --build-depends”
```

Comment:
Looks like the script repeats the same 7 lines three times, and the only difference is the extra argument (none, or `allow-newer`, or `allow-older`). Could you try to refactor the identical part into a separate function in this file?

Reply:
Done